src/components/shared/footer.tsx

```diff
@@ -82,6 +83,23 @@ interface FooterCcpaProps {
     isCcpa: boolean;
 }
 
-const FooterCcpa: FC<FooterCcpaProps> = ({ isCcpa }) => <Footer>{footerContents(isCcpa)}</Footer>
+const renderContent = (ccpaStatus: boolean): JSX.Element => {
+    if (!ccpaStatus){
+        return <a css={anchor} href="https://www.theguardian.com/help/privacy-policy">Privacy Settings</a>
+    } else {
+        return <a css={anchor} href="https://www.theguardian.com/help/privacy-policy">California Residents - Do Not Sell</a>
+    }
+}
+
+
+const FooterCcpa: FC<FooterCcpaProps> = ({ isCcpa }) => {
+    return <div css={container}>
+        &#169;
+        2020 Guardian News and Media Limited or its affiliated companies. All rights reserved.<br/>
```

Comment:
Can we make this 2020 dynamic to avoid us a job in January? 😀

Reply:
new date () added 👍